src/components/views/rooms/RoomHeader.tsx

```diff
@@ -150,7 +126,7 @@ export default function RoomHeader({ room }: { room: Room }): JSX.Element {
                         disabled={!!voiceCallDisabledReason}
                         title={!voiceCallDisabledReason ? _t("Voice call") : voiceCallDisabledReason!}
                         onClick={async () => {
```

Comment:
Just spotted the `async` here. The function is not async and it could be removed.